site/app/templates/grading/UploadImagesForm.twig

```diff
@@ -4,9 +4,9 @@
 {% block body %}
     <div class="flex-col flex-col-space">
         <p>
-            Instructions to download images of students:
+            <strong>Instructions to upload images of students:</strong>
```

Comment:
Why is this text necessary anyway?  Would it make sense to just put the help icon next to the text below this?

Reply:
I think this text should be there, sometimes users don't get focused on the help icon.